range_tombstone_list.cc

```diff
@@ -373,15 +373,24 @@ range_tombstone_list::reverter::insert(range_tombstones_type::iterator it, range
     return _dst._tombstones.insert_before(it, new_rt);
 }
 
+// Reserves space for at least one element so that subsequent v.push_back()
+// does not throw if element construction does not throw.
+// Amortizes space expansion so that a series of N calls to amortized_reserve()
+// starting from an empty container takes O(N) time.
+template <typename T>
```

Comment:
Please add a constraint (or open code it). Plain templates that give no hint about the input should be avoided.

Reply:
I'll just use chunked_vector<> then, it's not generically used yet.